### DISCUSSION TITLE
Problem with git pages (security error)

### DISCUSSION THREAD

**Discussion by mkaldev08** [community/community#31670] [New to GitHub] [answered]
hi guys, i made a netflix clone project to improve my skills and put it on git pages to show my teacher, only unfortunately my **git pages** 
![Captura de Ecrã (13)](https://user-images.githubusercontent.com/109039442/188029692-537cb92e-c071-472a-b42f-adb760b2bbea.png)
was blocked for detecting fraud.
what should i do to resolve the situation?

**Comment by radar-blip** [comment, marked as answer]
See marked answer here: #30261
Looks like the same issue.

**Reply by mkaldev08** [reply]
Thanks to help me!
I will go to Look that issue.

**Reply by radar-blip** [reply]
You're welcome. Hope you are able to fix it.
